tuxemon/resources/maps/bedroom_test.tmx

```diff
@@ -61,17 +61,6 @@
     <property name="cond3" value="is button_pressed K_RETURN"/>
    </properties>
   </object>
-  <object id="11" name="Get Fruitera" type="interact" x="144" y="112" width="16" height="16">
```

Comment:
It wouldn't make sense to have it as usually you get your first Pokemon (in this case, Tuxemon) from the professor. Besides, if you leave that in, there is no reason to talk to the professor in the first place and follow the story.